src/com/company/multiThreading/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.company.multiThreading;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+
+        DemoThread demoThread1 = new DemoThread();
+        Thread thread1 = new Thread(demoThread1);
+        thread1.start();
+        DemoThread demoThread2 = new DemoThread();
+        Thread thread2 = new Thread(demoThread2);
+        thread2.start();
+        thread1.join();
+        thread2.join();
+
+        try {
+            for (int i =0; i <5 ; i++){
+                System.out.println(Thread.currentThread().getName()+" : "+i);
+                Thread.sleep(1000);
```

Comment:
@Muhib-ur-Rehman You should **not** be using **sleep()**!
Please read task instruction carefully

Reply:
Oh sorry, I will correct this.